pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -118,10 +118,20 @@ private static void SetCulture(string cultureName)
 
         public static readonly Argument ARG_IN = new DocArgument(@"in", PATH_TO_DOCUMENT,
             (c, p) => c.SkylineFile = p.ValueFullPath);
-        public static readonly Argument ARG_SAVE = new DocArgument(@"save", (c, p) => c.Saving = true);
+        public static readonly Argument ARG_SAVE = new DocArgument(@"save", (c, p) =>
+        {
+            if (!c.Minimizing)
+                c.Saving = true;
+        });
         public static readonly Argument ARG_SAVE_SETTINGS = new DocArgument(@"save-settings", (c, p) => c.SaveSettings = true);
         public static readonly Argument ARG_OUT = new DocArgument(@"out", PATH_TO_DOCUMENT,
-            (c, p) => c.SaveFile = p.ValueFullPath);
+            (c, p) =>
+            {
+                if (!c.Minimizing)
```

Comment:
Would prefer to see this logic handled elsewhere.

Reply:
Moved to ValidateMinimizeResultsArgs